db/migrate/20210128001309_create_employers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEmployers < ActiveRecord::Migration[6.0]
```

Comment:
It's a good practice keeping different concerns in different commits. For example:
1. Commit where we create all related to the db table we need for our future model
2. Functionality - Models, Controllers, Views. Of course, if we need to create a module with a lot of complexity. Worth isolate it too. But it depends...